### PR TITLE
Add jdk8 tag suffix for existing JDK 8 images

### PR DESCRIPTION
**Pull request by MarkEWaite**
## Add JDK 8 tag for existing images

First step of JEP to implement JDK 11 as default JDK on Docker images

### CODE REVIEW

File: .ci/publish-manifests.sh [hunk 1]
```diff
@@ -106,6 +106,7 @@ publish-variant() {
 publish-alpine() {
     local archs="arm64 s390x ppc64le amd64"
     publish-variant "alpine"  "${archs}"  "alpine"
+    publish-variant "alpine"  "${archs}"  "alpine-jdk8"
```
Comment (timja) [inline .ci/publish-manifests.sh:109]: This script is not used

File: .ci/publish-tags.sh [hunk 2]
```diff
@@ -126,12 +126,14 @@ publish-alpine() {
     local version=$1
     local archs="arm64 s390x ppc64le amd64"
     publish-variant "${version}"  "alpine"  "${archs}" "alpine"
+    publish-variant "${version}"  "alpine"  "${archs}" "alpine-jdk8"
```
Comment (timja) [inline .ci/publish-tags.sh:129]: Script not used

**Comment by timja** [comment]
see https://github.com/jenkinsci/docker/pull/1174

**Review by timja — APPROVED** [review]
changes to docker-bake.hcl are fine, I recommend we merge https://github.com/jenkinsci/docker/pull/1174 first and then drop the changes to that script in this PR

**Review by slide — APPROVED** [review]

File: .ci/publish-manifests.sh (hunk 1 above)
Comment (MarkEWaite) [reply .ci/publish-manifests.sh]: Thanks!  Removed in 96a939d42493319f0693c153348c60a02dbd3ed1

File: .ci/publish-tags.sh (hunk 2 above)
Comment (MarkEWaite) [reply .ci/publish-tags.sh]: Removed in 96a939d42493319f0693c153348c60a02dbd3ed1

**Comment by MarkEWaite** [comment]
If others prefer, I can also rebase to remove the unnecessary commit from the history

**Comment by slide** [comment]
We can squash and merge so I wouldn't worry

**Pull request merged**